app/components/SearchHistoryItem/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { SearchHistoryItem } from './styles';
+
+const SearchHistory = ({ url, text }) => (
```

Comment:
Component name should be `SearchHistoryItem`

Reply:
Done!